### PR TITLE
Merge changes in dotnet/extensions from main to dev

### PR DESCRIPTION
**Pull request by joperezr**
Automatically generate PRs to merge changes from main to dev branch for dotnet/extensions.

### CODE REVIEW

**Comment by joperezr** [comment]
cc @mmitche

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -828,6 +828,23 @@
...
       }
     },
+    // Automate opening PRs to merge extensions main branch back to dev
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/extensions/blob/main//**/*"
+      ],
+      "action": "github-dnceng-branch-merge-pr-generator",
+      "actionArguments": {
+        "vsoSourceBranch": "dev",
```
Comment (riarenas) [inline Maestro/subscriptions.json:838]: This is for the branch that the [branch-merge-pr-generation-pipeline] should pull its yaml from. This should be main I think?
```suggestion
        "vsoSourceBranch": "main",
```

Comment (joperezr) [reply Maestro/subscriptions.json:838]: Oh, great catch, yeah I'm guessing main is right.

**Review by riarenas — APPROVED** [review]

**Comment by joperezr** [comment]
Thanks @riarenas. I'd appreciate if you could help merging as I don't have write access to this repo 😃

**Pull request merged**